src/App.css

```diff
@@ -34,10 +34,6 @@
   color: white;
 }
 
-.App-link {
-  color: #61dafb;
```

Comment:
Removed this style as i didnt see a usage for it and the color does not meet any of our new brand colors. Link styles should go in our link component